src/parser/jobs/ast/modules/Draw.tsx

```diff
@@ -256,28 +267,28 @@ export default class Draw extends Module {
 			SUGGESTION: Didn't use sleeve draw enough
 		*/
 		const sleevesMissed = Math.floor(this.sleeveTotalDrift / this.data.actions.SLEEVE_DRAW.cooldown)
-		if (this.sleeveUses > 0 && sleevesMissed > 0) {
+		if (this.sleevesUsed > 0 && sleevesMissed > 0) {
```

Comment:
This is a tiered suggestion, so you can invert this conditional where you check if sleeves used are zero first, else use this block instead. Sleeves missed will be checked by the value param anyway, so sleeves used will be caught by the original condition for the other suggestion. Pseudo code but:

```ts
if (sleevesUsed === 0) {
  // do the no sleeves suggestion
} else {
  // do the tiered suggestion for misses without any additional condition checks
}
```

Reply:
updated this and the other section in the same format